src/components/tooltip/Tooltip.css

```diff
@@ -90,6 +90,11 @@
   color: var(--grey-50);
 }
 
+.tooltipDetailsDescription {
+  /* Stop long descriptions from making the tooltip disruptively long. */
+  max-width: var(--tooltip-detail-max-width);
+}
+
```

Comment:
Initially I was wondering if  this `max-width` shouldn't be on `.tooltip` directly. But I see that this var isn't available on `.tooltip`, because it's set on the content.